lrpar/README.md

```diff
@@ -2,10 +2,16 @@
 
 `lrpar` provides a Yacc-compatible parser (where grammars can be generated at
 compile-time or run-time). It can take in traditional `.y` files and convert
-them into an idiomatic Rust parser. More details can be found in the [grmtools
-book](https://softdevteam.github.io/grmtools/master/book); the
-[quickstart guide](https://softdevteam.github.io/grmtools/master/book/quickstart.html)
-is a good place to start.
+them into an idiomatic Rust parser. A good place to start is the "quick start
+guide"; the "grmtools book" and API have more detailed information:
```

Comment:
One niggle. If we are using `;` to delimit list entries, then there's one missing before "and API".

I'd also `s/API/the API reference/` since the other entries start with `the`.

Maybe remove double quoting? up to you.

Reply:
See if dce7a99 works for you?